HospitalGame_Business/src/business/Data/DataRoom.java

```diff
@@ -16,40 +16,52 @@
  *
  * @author andreasmolgaard-andersen
  */
-public class DataRoom implements IRoom{
+public class DataRoom implements IRoom {
+
     private String name;
     private HashMap<String, IRoom> exits;
     private boolean inspected;
+    private IRoom exit;
+    private Set<Directions> exitDirections;
```

Comment:
IRoom exit and exitDirections does make sence since it should be taken from the HashMap